protocols/v2/roles-logic-sv2/src/handlers/mining.rs

```diff
@@ -247,6 +247,7 @@ pub trait ParseUpstreamMiningMessages<
 
         // Is fine to unwrap on safe_lock
         match (message_type, payload).try_into() {
+            // Can Panic
```

Comment:
Not sure why do you want communicate with this comment, can you expand it?
Also note that if the `Mutex` that hold the routing logic (or Self) is poisoned, is very likely that we are in a non recoverable situation. 

This review apply also to all the other same comment in this file.

Reply:
So when I was looking for possible panics throughout this match to see what we have to look out for - so these comments just specify which of these matches can panic. This was just my way of keeping track of which ones I had investigated - they can be removed if you don't think they provide any value.

Reply:
Yep I think that we can remove them as it explicitly said in the code so no need for a comment IMO